### ISSUE TITLE
Direct E2E support requests to help documentation

### ISSUE THREAD

**Issue by JoeTice** [department-of-veterans-affairs/va.gov-team#42737]
## Issue Description
VFS teams commonly create support requests as a result of failed Cypress test executions. In many cases, there are a few techniques that the VFS team can apply themselves in order to remedy the issue. Let's explore the idea of automating a link to Cypress troubleshooting documentation hosted on the Platform Website when someone opens a support request assigned to our team and mentioning Cypress failures.

---
## Tasks
- [ ] Research the possibility of adding capability to the support bot
- [ ] Extend the support bot to scan for the keyword 'Cypress' and perhaps others in the text of the support request
- [ ] If the support request refers to Cypress failures then it will respond with a canned message linking to the Platform Website document for troubleshooting Cypress failures

## Acceptance Criteria
- [ ] The support bot detects references to Cypress failures in support requests
- [ ] The support bot responds to requests about Cypress failures with a troubleshooting document encouraging teams to resolve the issue themselves if they can

**Comment by holdenhinkle** [comment]
Here's the repo for the Slack Platform Support feature - https://github.com/department-of-veterans-affairs/platform-support-slackbot

**Comment by holdenhinkle** [comment]
I thoroughly reviewed the code and have a solid plan to quickly code this up.

I met with Peter to discuss my approach, and some other ideas I had that could be helpful to VFS and VSP teams.

There's quite a bit of configuration needed to get this running locally:
- https://vfs.atlassian.net/wiki/spaces/SRE/pages/1883406342/Platform+Support+Bot
- https://vfs.atlassian.net/wiki/spaces/SRE/pages/1901559843/Using+Platform+Support+Bot+locally
- https://vfs.atlassian.net/wiki/spaces/SRE/pages/1883340805/Platform+Support+Slack+Bot+Configuration

I'm waiting on access to a Google Sheet that I need to copy so I can develop and test changes to the bot locally. I just sent another request to whoever owns the document, this time with a more detailed message:

```
Hello,
I’m working on this ticket to make an update to the Slack Support Bot on DS@VA and need access to this sheet so I can make a copy of it.
Ticket - https://app.zenhub.com/workspaces/planning-board-qa-standards-612411db8ec47700127369e2/issues/department-of-veterans-affairs/va.gov-team/42737
I don’t need the data that’s been collected, only the column headers, etc. so I can develop locally against an empty copy.
Thanks,
Holden Hinkle
QA Standards Team
Oddball, Inc.
```

**Comment by powellkerry** [comment]
This may duplicate efforts on the SRE team to auto-answer support requests: https://github.com/department-of-veterans-affairs/va.gov-team/issues/36853

We currently have a POC that we are testing.

**Comment by pjhill** [comment]
We should be able to pick up a discussion with SRE team this week.

**Comment by holdenhinkle** [comment]
I reached out to Kerry Powell to find out what the status of the Support Request Auto Answer Service is. I'm awaiting his response.

**Comment by holdenhinkle** [comment]
Kerry Powell
  3:20 PM
Initial development is complete, though we are reaching out to team leads to refine both the approach and the documentation that will initially be used to auto-answer support requests. You can view the documentation here: 
https://vfs.atlassian.net/wiki/spaces/SRE/pages/2241396873/Automatic+Answers+and+Replies

Here is a link to the current mappings for answers: 
https://vfs.atlassian.net/wiki/spaces/SRE/pages/2260336690/Auto-Answer+Mappings

**Comment by holdenhinkle** [comment]
Moving this this to the product backlog for now until the SRE team completes the MVP.

**Comment by pjhill** [comment]
Given that the SRE team created the automation to handle this workflow and that the original scope of the ticket was to create that automation, I'm going to close this ticket. 

In the future, we will contribute to the SRE team's automation under a separate ticket.
